src/main/java/be/syntra/devshop/DevshopBack/repositories/ProductRepository.java

```diff
@@ -1,11 +1,15 @@
 package be.syntra.devshop.DevshopBack.repositories;
 
 import be.syntra.devshop.DevshopBack.entities.Product;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
```

Comment:
when committing, you can check the boxes "optimize imports" && "reformat code". Every class you touched will do an automatic ctrl-alt-O and ctrl-alt-L. 
I noticed, because Sort isn't used anywhere, and no spaces after commas :)

Reply:
i did this for front end and optimize imports is currently checked, shouldn't be happening
i did misclicked the reformatting it seems....

Reply:
haha, ctrl-alt-L nog :)

Reply:
done, on the second try